### ISSUE TITLE
Include 'Kitchen Sink' markup in pattern library

### ISSUE THREAD

**Issue by mattheu** [humanmade/kalutara#23]
Include a pattern library page with 'Kitchen Sink' type HTML for general styling. Headings, lists, paragraphs etc. 

Could even take this further and have core Gutenberg blocks and other WordPress patterns?

It doesn't really make sense for me to add dummy template parts that will never really be used for this purpose. 

Could be filterable so people can add their own patterns?

**Comment by goldenapples** [comment]
Yeah, I don't know what would be the best way of doing this. If you're just looking for ways to display content styles, maybe including a KSS parser (https://github.com/kss-php/kss-php) in this plugin and defining a way to point to the sass files that should be parsed would be good?
